service/kas/access/provider.go

```diff
@@ -73,6 +75,36 @@ func (p *Provider) IsReady(ctx context.Context) error {
 	return nil
 }
 
+// ApplyConfig stores the latest KAS configuration, tracks the associated security
+// overrides, and emits a warning when the configured clock skew exceeds the default.
+func (p *Provider) ApplyConfig(cfg KASConfig, securityCfg *config.SecurityConfig) {
+	p.KASConfig = cfg
+	p.securityConfig = securityCfg
+
+	if p.Logger != nil {
+		if skew := p.acceptableSkew(); skew > config.DefaultUnsafeClockSkew {
+			p.Logger.Warn("configured SRT acceptable skew exceeds default",
+				slog.Duration("configured_skew", skew),
+				slog.Duration("default_skew", config.DefaultUnsafeClockSkew),
```

Comment:
IIRC don't tools like newrelic and datadog let you group things using a dot separator? Grouping them like this may help when using a logs aggregator
```suggestion
				slog.Duration("cfg.skew.value", skew),
				slog.Duration("cfg.skew.default", config.DefaultUnsafeClockSkew),
```

Reply:
It has been a while so I am not sure if this is true. @gemini-code-assist thoughts on this?